### PR TITLE
Update clarity-billboard.md

### PR DESCRIPTION
**Pull request by kauly**
## Description

The Billboards smart contract tutorial has some typos that can cause confusion to newcomers.

## Checklist

- [ ] [Conventional commits were used](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] New links to files and images were verified
- [ ] For fixes/refactors: all existing references were identified and replaced
- [ ] [Style guide](https://developers.google.com/style) was reviewed and applied
- [ ] Clear code samples were provided
- [ ] People were tagged for review

### CODE REVIEW

File: docs/tutorials/clarity-billboard.md [hunk 1]
```diff
@@ -284,8 +284,9 @@ Clarinet.test({
 
     let [event] = block.receipts[0].events;
     let { sender, recipient, amount } = event.stx_transfer_event;
-    sender.expectPrincipal("ST1J4G6RR643BCG8G8SR6M2D9Z9KXT2NJDRK3FBTK");
+    sender.expectPrincipal(wallet_1.address);
     recipient.expectPrincipal(
+    // you can get the contract address using clarinet console
       "ST1HTBVD3JG9C05J7HBJTHGR0GGW7KXW28M5JS8QE.billboard"
     );
```
Comment (obycode) [inline docs/tutorials/clarity-billboard.md:291]:
```suggestion
    recipient.expectPrincipal(`${deployer.address}.billboard`);
```

**Review by obycode — CHANGES_REQUESTED** [review]
Thanks for pointing this out. I agree it is unclear. What do you think about this additional improvement?

File: docs/tutorials/clarity-billboard.md (hunk 1 above)
Comment (kauly) [reply docs/tutorials/clarity-billboard.md:291]: Much better, thanks

File: docs/tutorials/clarity-billboard.md [hunk 2]
```diff
@@ -284,10 +284,8 @@ Clarinet.test({
 
     let [event] = block.receipts[0].events;
     let { sender, recipient, amount } = event.stx_transfer_event;
-    sender.expectPrincipal("ST1J4G6RR643BCG8G8SR6M2D9Z9KXT2NJDRK3FBTK");
-    recipient.expectPrincipal(
-      "ST1HTBVD3JG9C05J7HBJTHGR0GGW7KXW28M5JS8QE.billboard"
-    );
+    sender.expectPrincipal(wallet_1.address);
+    recipient.expectPrincipal(`${deployer.address}.billboard`);
```
Comment (obycode) [inline docs/tutorials/clarity-billboard.md:288]: Almost forgot, this also requires an addition at the top of the test, to define `deployer`.

```ts
        let deployer = accounts.get('deployer')!;
```

Comment (kauly) [reply docs/tutorials/clarity-billboard.md:288]: yea, I saw that you but this in another pr tho

Comment (obycode) [reply docs/tutorials/clarity-billboard.md:288]: Right, I made the matching changes in the example in the clarinet repo 👍

**Review by obycode — APPROVED** [review]
Thanks!

**Pull request merged**